help/en/package/jmri/jmrix/openlcb/swing/networktree/NetworkTreePane.shtml

```diff
@@ -64,7 +64,18 @@
           <td>You can fill that out and click "Write" to configure the node. The top few fields are
           used to give it a user-chosen name and description, so you can remember this node later.
           You can set the events that correspond to different inputs and outputs, and even give
-          those user-convenient names. That's all stored permanently in the node.</td>
+          those user-convenient names. That's all stored permanently in the node.
+          <p>
+          As you change values, you'll notice the background on some type of items and tabs
+          change from white to orange.  Orange indicates that you've changed a
+          value and should write it to the node so it's not lost. If you enter
+          an invalid value, such as an integer value that's too large or too
+          small for that particular item, the background will turn to blue.
```

Comment:
Are these colors due to color blindness? I'm not sure, but I think DecoderPro has the colors yellow and red. The color red usually stands for "stop"or "error", for example in traffic lights both for railroad and roads.

It's not up to me to decide what colors should be used in this case, but I don't think I have ever seen blue as error.